oci/Containerfile

```diff
@@ -15,22 +15,18 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ghcr.io/crc-org/crc-extension-builder:latest AS builder
-
-COPY . .
-
-RUN yarn --network-timeout 180000 \
-  && yarn build
-
-FROM scratch
+FROM scratch as builder
+COPY package.json /extension/
+COPY LICENSE /extension/
+COPY icon.png /extension/
+COPY README.md /extension/
+COPY dist /extension/dist
```

Comment:
I think we should keep multi layered container file to build extension and remove the build itself from release.yaml

Reply:
This is to be aligned with the way other extensions are built. Cleanup will be done after release

Reply:
AFAIK these files are here so it can be built in Konflux
so I guess that removing them will make the build fail

cc @adrianriobo

Reply:
restored multistage build, tested on fork.